app/main/controllers/template/RTMediaTemplate.php

```diff
@@ -459,12 +459,33 @@ public function media_update_success_messege() {
 
 	/**
 	 * Show error message for media update error.
+	 *
+	 * @return void
 	 */
 	public function media_update_success_error() {
-		$message = apply_filters( 'rtmedia_update_media_message', esc_html__( 'Error in updating Media', 'buddypress-media' ), true );
-		$html    = "<div class='rtmedia-error  media-edit-messge'>" . esc_html( $message ) . '</div>';
+		/**
+		 * To filter error message on media update.
+		 *
+		 * @param string $message Raw error message.
+		 * @param bool   $bool
```

Comment:
Can we have description for bool parameter as well, here and bottom as well.

Reply:
@ravatparmar Please check PR description, there I've explained this.